src/calendar.js

```diff
@@ -58,13 +58,32 @@ export default class Calendar extends Emitter {
       this.locale           = locales[this.opts.lang];
       this.format           = this.opts.formatDate || this.locale.formatDate || '%D';
       this.opts.startOfWeek = this.opts.startOfWeek || this.locale.startOfWeek;
+
+      let { currentDate } = this.opts;
+      currentDate = currentDate.toLocaleString('en', {
+        year: 'numeric',
+        month: '2-digit',
+        day: '2-digit'
+      }).split('/');
+      const year  = parseInt(currentDate[2]);
+      const month = parseInt(currentDate[0]) - 1;
```

Comment:
I see this piece of code had just been added, so let's do what we talked about and use native Date functions:
```suggestion
      const year  = currentDate.getFullYear();
      const month = currentDate.getMonth();
```